packages/flutter_goldens_client/lib/client.dart

```diff
@@ -10,17 +10,17 @@ import 'package:file/local.dart';
 import 'package:platform/platform.dart';
 import 'package:process/process.dart';
 
+import 'skia_client.dart';
+
 // If you are here trying to figure out how to use golden files in the Flutter
 // repo itself, consider reading this wiki page:
 // https://github.com/flutter/flutter/wiki/Writing-a-golden-file-test-for-package%3Aflutter
 
 const String _kFlutterRootKey = 'FLUTTER_ROOT';
 
-/// A class that represents a clone of the https://github.com/flutter/goldens
-/// repository, nested within the `bin/cache` directory of the caller's Flutter
-/// repository.
+/// A base class that provides shared information to the [SkiaGoldClient] and
+/// [GoldensRepositoryClient].
```

Comment:
Should this one be abstract then?

Reply:
You never seem to instantiate a GoldensClient, though. Only its subclasses get instantiated, so you should be able to still mark this class as abstract.

Reply:
It is instantiated once on line#215 of flutter_goldens.dart.